### ISSUE TITLE
Exited with code 1: ERROR! input is already encryptedm when decrypting vault file

### ISSUE THREAD

**Issue by wheelerlaw** [timo-reymann/idea-ansible-vault-integration#8]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Used versions

```
$ ansible-vault --version
ansible-vault 2.9.23
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/wlaw/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  executable location = /usr/bin/ansible-vault
  python version = 3.9.5 (default, May 14 2021, 00:00:00) [GCC 11.1.1 20210428 (Red Hat 11.1.1-1)]
```
* Host OS: Linux

## Problem description
I am trying to decrypt an ansible vault file. 

## Expected Behavior

I want to decrypt a whole ansible vault file. I have the `ANSIBLE_VAULT_PASSWORD_FILE` environment variable defined in my `~/.profile` file pointing to the location of a file with the vault password in it. I should be able to decrypt the whole file. 

### Actual Behavior

I get an exception, file is not decrypted. 

### Steps to Reproduce

I open up the `vault.yml` file with IntelliJ. It shows the ciphertext in the editor:
![image](https://user-images.githubusercontent.com/3081566/125012304-f8631c00-e037-11eb-91a7-38980b4607b0.png)

I go to decrypt it by doing Alt + Enter. The only option available is "Vault ansible secret", which right off the bat, is incorrect behavior because this file is already encrypted. 

![image](https://user-images.githubusercontent.com/3081566/125012863-e03fcc80-e038-11eb-8b95-0de816529942.png)

If I select that, I get an exception:

```
de.timo_reymann.ansible_vault_integration.action.execution.AnsibleVaultWrapperCallFailedException: Exited with code 1: ERROR! input is already encrypted
	at de.timo_reymann.ansible_vault_integration.action.vaultaction.AnsibleVaultAction.executeCommand(AnsibleVaultAction.kt:85)
	at de.timo_reymann.ansible_vault_integration.action.vaultaction.AnsibleVaultAction.execute(AnsibleVaultAction.kt:44)
	at de.timo_reymann.ansible_vault_integration.action.vaultaction.AnsibleVaultEncryptAction.execute(AnsibleVaultEncryptAction.kt:20)
	at de.timo_reymann.ansible_vault_integration.action.runnable.EncryptAnsibleVaultRunnable.run(EncryptAnsibleVaultRunnable.kt:19)
	at de.timo_reymann.ansible_vault_integration.action.execution.AnsibleVaultTask.run(AnsibleVaultTask.kt:26)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:998)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:497)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:228)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:228)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

### References

* #0000

**Comment by timo-reymann** [comment]
Hello @wheelerlaw  and thank you for your bug report,

this is a mixture of an feature request and a bug report.

The plugin currently has no support for entirely encrypted yaml files. The bug here is that it shouldnt show you the possibility to encrypt an encrypted string.
For the first part it is the case that this totally makes sense and simply is not implemented yet, as there were no need for it, till now.

Proposed solution:
- Hide the context action for encrypted strings
- Provide a context/"right click"-action on file level for entirely encrypted files

As file context actions are a bit new for me to implement the feature might take a while, where the bug might be solved a bit earlier.

**Comment by timo-reymann** [comment]
Alright so the fix part will be available with the next release. The feature part is currently under testing :)

**Comment by timo-reymann** [comment]
Will be available with 2.1.0 in the marketplace.

Tested on MacOS and Ubuntu.

If you encounter some problems after the update related to this, please feel free to comment on this issue and reopen it :)
